ansible/roles/stack-sunbird/templates/sunbird_player.env

```diff
@@ -163,4 +163,7 @@ sunbird_azure_storage_account_name=https://{{sunbird_public_storage_account_name
 
 #Release-4.1.0
 sunbird_google_oauth_ios_clientId={{sunbird_google_oauth_ios_clientId | default("")}}
-sunbird_google_oauth_ios_clientSecret={{sunbird_google_oauth_ios_clientSecret | default("")}}
\ No newline at end of file
+sunbird_google_oauth_ios_clientSecret={{sunbird_google_oauth_ios_clientSecret | default("")}}
+
+#Release-4.2.0
+crypto_encryption_key_external={{crypto_encryption_key_external}}
```

Comment:
What’s the default value ? Or it doesn’t require a default is a mandatory variable?